docs/runbooks/production-rollback.md

```
@@ -0,0 +1,174 @@
+
+1. **Stop Application Traffic:**
+   ```bash
+   # Scale backend replicas to 0 during database recovery
+   docker service scale profiletailors-smp_backend=0
+   ```
+
+2. **Point-In-Time Restore (PITR) from Backup:**
+   If schema or data corruption occurred, restore PostgreSQL from the latest verified automated backup prior to the deployment timestamp. Refer to [`docs/infrastructure/private-beta-backup-restore-status.md`](../infrastructure/private-beta-backup-restore-status.md) for snapshot location and restore procedures.
+
+3. **Re-apply Safe Revision:**
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)